src/include/ipc4/pipeline.h

```diff
@@ -105,7 +105,8 @@ struct ipc4_pipeline_create {
 			uint32_t lp             : 1;
 			uint32_t rsvd1          : 3;
 			uint32_t attributes     : 16;
-			uint32_t rsvd2          : 10;
+			uint32_t core_id        : 4;
+			uint32_t rsvd2          : 6;
```

Comment:
@plbossart @ujfalusi - fyi driver will need a matching update, which may needed connected into core refcount/PM logic ?

Reply:
and @ranj063  and @bardliao  as well. It looks straightforward to add as we do the same thing for IPC3 and the CORE_ID token is defined in topologies.